src/models/index.js

```diff
@@ -0,0 +1,19 @@
+const Sequelize = require('sequelize');
+const CatModel = require('./cats');
+
+const setUpDatabase = () => {
+    const connection = new Sequelize("have_i_fed_the_cat_app", "postgres", "password", {
+    host: "localhost",
```

Comment:
You have used single quotations throughout the project so should be consistent and use them everywhere
```
    const connection = new Sequelize('have_i_fed_the_cat_app', 'postgres', 'password', {
    host: 'localhost',
```